.golangci.yaml

```diff
@@ -1,5 +1,4 @@
 run:
-  deadline: 15m
```

Comment:
Unsupported, we have configured timeout both in workflow and in the `make lint` target